platform/deploy.go

```diff
@@ -25,6 +26,7 @@ type Config struct {
 	DockerEncodedAuth string             `hcl:"docker_encoded_auth,optional"`
 	Domain            string             `hcl:"domain"`
 	Env               *map[string]string `hcl:"env"`
+	EnvFromFile       *string            `hcl:"envFromFile"`
```

Comment:
we probably should decide if we wanna go for the *string syntax (so the type is nillable) or if we want to define such things as non-nillable, and supply "optional" in the HCL definition (e.g. how it is done for DockerEncodedAuth on L26). I don't really care about either, but we should really stick to one and not mix those two up.